src/main/java/bio/terra/common/flagsmith/FlagsmithClient.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.common.flagsmith;
+
```

Comment:
i don't understand why you are adding this. there is a flagsmithService that's doing the exact same thing already.

Reply:
This is the bare minimum needed for the client, the service adds caching and many other features 

We don't want caching, hence this file

Reply:
i'm blocking the change because I think we need general consensus on caching. everywhere else is doing caching including UI.

Reply:
if you want, you can make the caching time configurable. but there's no need to *duplicate* code

Reply:
I will set up a meeting, this PR does not change existing behavior in WSM/UI/Other and blocking a change is not needed. again we should not cache access information. 
I will set up a meeting if needed to resolve this. let me know.

Reply:
Thanks for meeting, As discussed - reduced the cache expiry. Same client API key available in CLI can be used with service as well. hence redid the PR